crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -123,38 +123,6 @@ pub struct CheckpointResponse {
     pub detail: Option<CheckpointContents>,
 }
 
-impl CheckpointResponse {
```

Comment:
Is this refactoring related to this PR and epoch DB?

Reply:
Yes I had to move this check into within SafeClient due to the use of the epoch db.